src/libOpenImageIO/color_ocio.cpp

```diff
@@ -1848,7 +1876,7 @@ bool
 ImageBufAlgo::ociodisplay(ImageBuf& dst, const ImageBuf& src,
                           string_view display, string_view view,
                           string_view from, string_view looks, bool unpremult,
-                          string_view key, string_view value,
+                          bool inverse, string_view key, string_view value,
                           ColorConfig* colorconfig, ROI roi, int nthreads)
 {
```

Comment:
Don't know how to select the line 1872, but don't you want to pass inverse there too?

Reply:
yes, I think you're right